app/main/actions/exec.js

```diff
@@ -1,8 +1,8 @@
 const { fork } = require('child_process');
 
 module.exports = (message, arg) => {
-  if (arg) {
-    fork(arg, {
+  if (arg.script) {
```

Comment:
Good enough argument to move forward with your changes. 👍

We'll need to update the core screensaver plugin to this new struct.